static/src/javascripts-legacy/projects/common/modules/experiments/tests/recommended-for-you.js

```diff
@@ -38,12 +38,12 @@ define([
     return function () {
         this.id = 'RecommendedForYouRecommendations';
         this.start = '2017-01-17';
-        this.expiry = '2017-03-08';
+        this.expiry = '2017-04-05';
```

Comment:
Not an issue, but it is no longer necessary to have the expiries off-by-one:

Expiries (Scala switch, JS test) now have the same behaviour; the JavaScript test will be able to run until 23:59 on 2017-04-05